packages/prisma-client/prisma/schema.prisma

```diff
@@ -124,3 +128,16 @@ view DashboardProject {
   isDeleted   Boolean  @default(false)
   isPublished Boolean
 }
+
+model CustomDomain {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+
+  domain String @unique
+
+  user   User?   @relation(fields: [userId], references: [id])
+  userId String?
+
+  Project   Project? @relation(fields: [projectId], references: [id])
+  projectId String?
```

Comment:
The only reason is clean up, right? Delete all custom domains without references from cloudflare. But this can be done 
Select all custom domains - find all not in our db - delete